capplet/gsm-properties-dialog.c

```diff
@@ -102,6 +102,8 @@ find_by_app (GtkTreeModel *model,
                         g_object_unref (iter_app);
                         return TRUE;
                 }
+
+                g_object_unref (iter_app);
```

Comment:
Isn't it already unref in line102?
But maybe i don't understand the code well....

Reply:
Currently, it's only unreferenced when both objects are equal before returning TRUE.
We should unreference all returned objects by gtk_tree_model_get.

> Returned values with type G_TYPE_OBJECT have to be unreferenced, values with type G_TYPE_STRING or G_TYPE_BOXED have to be freed. Other values are passed by value.

https://developer.gnome.org/gtk3/stable/GtkTreeModel.html#gtk-tree-model-get

Reply:
OK